app/controllers/orders/index.js

```diff
@@ -70,6 +74,10 @@ export default Ember.Controller.extend(SearchMixin, {
           this.afterSearch(results);
           return results;
         });
+    },
+
+    clearSearch() {
```

Comment:
same question for this action too.

Reply:
For the cross button which appears on the search tab